README.md

```diff
@@ -55,64 +55,60 @@ Web version can be found here: [DCA website](https://evalle.github.io/DCA/)
 - [Pull](https://docs.docker.com/engine/reference/commandline/pull/) and [delete](https://docs.docker.com/registry/spec/api/#deleting-an-image) images from a registry
 
 ### Domain 3: Installation and Configuration (15% of exam)
-- [Demonstrate the ability to upgrade the Docker engine](https://docs.docker.com/install/linux/docker-ce/ubuntu/#upgrade-docker-engine---community)
-- [Complete setup of repo, select a storage driver, and complete installation of Docker
-engine on multiple platforms](https://docs.docker.com/install/)
-- [Configure logging drivers (splunk, journald, etc)](https://docs.docker.com/config/containers/logging/configure/)
-- [Setup swarm, configure managers, add nodes, and setup backup schedule](https://docs.docker.com/engine/swarm/admin_guide/)
-- [Create and manage user and teams](https://docs.docker.com/datacenter/dtr/2.4/guides/admin/manage-users/create-and-manage-teams/)
-- [Interpret errors to troubleshoot installation issues without assistance](https://docs.docker.com/config/daemon/#troubleshoot-the-daemon)
-- [Outline the sizing requirements prior to installation](https://docs.docker.com/datacenter/ucp/2.2/guides/admin/install/system-requirements/#hardware-and-software-requirements)
-- [Understand namespaces, cgroups, and configuration of certificates](https://docs.docker.com/engine/docker-overview/#namespaces)
-- [Use certificate-based client-server authentication to ensure a Docker daemon has the
-rights to access images on a registry](https://docs.docker.com/engine/security/certificates/)
-- Consistently repeat steps to deploy Docker engine, UCP, and DTR on AWS and on
-premises in an HA config. [Docker,](https://docs.docker.com/install/linux/docker-ce/ubuntu/) [DTR,](https://docs.docker.com/datacenter/dtr/2.3/guides/admin/install/) [UCP,](https://docs.docker.com/ee/ucp/), [Docker on AWS](https://docs.docker.com/docker-for-aws/) and possibly [on premises HA config](https://docs.docker.com/engine/swarm/admin_guide/#add-manager-nodes-for-fault-tolerance)
-- [Complete configuration of backups for UCP and DTR](https://docs.docker.com/datacenter/ucp/2.2/guides/admin/backups-and-disaster-recovery/)
-- [Configure the Docker daemon to start on boot](https://docs.docker.com/install/linux/linux-postinstall/)
+- [Describe sizing requirements for installation](https://docs.docker.com/docker-for-windows/install/#system-requirements)
+- [Describe and demonstrate the setup of repo, selection of a    storage driver, and installationof the Docker engine on multiple platforms](https://docs.docker.com/engine/install/)
```

Comment:
Thanx a lot for the awesome work man!

Unfortunately I still find that some of the old links are now changed to Mirantis docs. 

I will check the typos and all the links, one by one, and I will create another PR. 
Because now I am on Domain 3 and I still find bad links. 

For now, I will close this one and I will be back in a couple of days. 

Stay safe!